File: C++/Test_RasBot.cpp
#include "RasBot.h"
#include <wiringPi.h>

int main(){

    RasBot robot;

    robot.moveF(80);
    delay(1000);
    robot.moveB(80);
    delay(1000);
    robot.turnL(50);
    delay(1000);
    robot.turnR(50);
    delay(1000);

    robot.stop();
    
    return 0;

}

File: Python/Test_RasBot.py
from RasBot import RasBot
from time import sleep

# Delete the "True" parameter if you don't want debug mensages from the class RasBot
robot = RasBot(True)

# The values to the methods must be between 0 - 100 (0% - 100%)
robot.moveF(80)
sleep(1)
robot.moveB(80)
sleep(1)
robot.turnL(50)
sleep(1)
robot.turnR(50)
sleep(1)
robot.stop()
